README.md

```diff
@@ -64,6 +64,10 @@ This driver supports only one topology key:
 `topology.gke.io/zone`
 that represents availability by zone (e.g. `us-central1-c`, etc.).
 
+### CSI Windows Support
+
+GCE PD driver starts to support CSI Windows (alpha) with version v1.0.*. It requires to install [CSI Proxy] (https://github.com/kubernetes-csi/csi-proxy) binary on each Windows node. The latest version of CSI Proxy is Beta. GCE PD driver start to support Beta CSI Proxy starting from v1.1.0. Starting Kubernetes 1.19, CSI Proxy binary is automatically installed during node start up. In Kubernetes 1.20, CSI Proxy will be running a windows service to improve its stablibity.
```

Comment:
I don't think we can make a lot of statements about csi proxy installation. People may not be using kube-up or GKE.

We probably should say that csi-proxy needs to be installed by the cluster distribution/admin on all nodes, and mention that it is installed with kube-up, and plans to be installed automatically by GKE.

Reply:
updated